### PR TITLE
Configs Applier -> master

### PR DESCRIPTION
**Pull request by greenblitz4590**

### CODE REVIEW

**Review by bossyshrimp68 — COMMENTED** [review]
re-request review when name is fixed

File: src/main/java/frc/utils/ctre/PhoenixProUtils.java [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+import java.util.function.Supplier;
+
 public class PhoenixProUtils {
 
     public static <T> StatusSignal<T> getRefreshedSignal(boolean refresh, StatusSignal<T> signal) {
         return refresh ? signal.refresh() : signal;
     }
 
-}
+    public static boolean checkAndRetry(Supplier<StatusCode> statusCodeSupplier, int tries){
```
Comment (bossyshrimp68) [inline src/main/java/frc/utils/ctre/PhoenixProUtils.java:14]:
```suggestion
    public static boolean checkAndRetry(Supplier<StatusCode> statusCodeSupplier, int numberOfTries){
```

**Review by bossyshrimp68 — CHANGES_REQUESTED** [review]

**Review by bossyshrimp68 — APPROVED** [review]
good job!!

File: src/main/java/frc/utils/ctre/PhoenixProUtils.java [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
 
+import java.util.function.Supplier;
+
 public class PhoenixProUtils {
 
     public static <T> StatusSignal<T> getRefreshedSignal(boolean refresh, StatusSignal<T> signal) {
         return refresh ? signal.refresh() : signal;
     }
 
+    public static boolean checkAndRetry(Supplier<StatusCode> statusCodeSupplier, int numberOfTries) {
```
Comment (tnrz135) [inline src/main/java/frc/utils/ctre/PhoenixProUtils.java:14]: This function name suggests it has  2 responsibilities, as it has "and" in its name, which is also not very clean and breaks the Single Responsibility Principle. But actually this function does only one thing and that is the retries. Consider a better name for the function.

**Review by tnrz135 — CHANGES_REQUESTED** [review]

**Review by bossyshrimp68 — APPROVED** [review]

**Pull request merged**